app/models/competition.rb

```diff
@@ -2509,12 +2509,23 @@ def payments_enabled?
     competition_payment_integrations.exists?
   end
 
+  def connected_payment_integration_types
+    raw_types = self.competition_payment_integrations.pluck(:connected_account_type)
+    raw_types.map { |type| CompetitionPaymentIntegration::AVAILABLE_INTEGRATIONS.invert[type] }
+  end
+
+  def payment_integration_connected?(integration_name)
+    CompetitionPaymentIntegration.validate_integration_name!(integration_name)
+
+    self.competition_payment_integrations.send(integration_name).exists?
+  end
+
   def stripe_connected?
```

Comment:
Do we still need these if we could just call `competition.payment_integration_connected?('stripe') and have a `.to_sym` sanitation?

Reply:
Actually, a symbol makes sense in this context because it expresses that whatever you're passing is a static/frozen constant. Strings are too "mutable", if you get what I mean

Reply:
Can add a sanitizer nonetheless